modules/ROOT/pages/_partials/commons/common-sgw-replication-monitor.adoc

```diff
@@ -67,4 +64,114 @@ include::{snippet-p2psync-ws}[tags=p2p-act-rep-status,indent=0]
 ====
 =====
 
+
+[#lbl-repl-states]
+=== Replication States
+<<tbl-states>> shows the different states, or activity levels, reported in the API; and the meaning of each.
+
+.Replicator activity levels
+[#tbl-states,cols="^1,4"]
+|===
+h|State
+h|Meaning
+
+|`STOPPED`
+|The replication is finished or hit a fatal error.
+
+|`OFFLINE`
+|The replicator is offline as the remote host is unreachable.
+
+|`CONNECTING`
+|The replicator is connecting to the remote host.
+
+|`IDLE`
+|The replication caught up with all the changes available from the server.
+The `IDLE` state is only used in continuous replications.
+
+|`BUSY`
+|The replication is actively transferring data.
+|===
+
+NOTE: The replication change object also has properties to track the progress (`change.status.completed` and `change.status.total`).
+But since the replication occurs in batches and the total count can vary through the course of a replication, those progress indicators are not very useful from the standpoint of an app user.
+Hence, these should not be used for tracking the actual progress of the replication.
```

Comment:
Not sure if this sentence is necessary but this is interesting so it's something we may need to improve.

Reply:
Changed in line with comment below.